internal/cypress/config.go

```diff
@@ -20,7 +19,7 @@ type Project struct {
 	config.TypeDef `yaml:",inline"`
 	Defaults       config.Defaults `yaml:"defaults" json:"defaults"`
 	ShowConsoleLog bool
-	RawConfig      string             `yaml:"-" json:"-"`
+	RawConfigFile  string             `yaml:"-" json:"-"`
```

Comment:
Maybe naming it straight up `ConfigFile` is self explanatory enough.
```suggestion
	ConfigFile  string             `yaml:"-" json:"-"`
```

or

```suggestion
	ConfigFilepath  string             `yaml:"-" json:"-"`
```

Reply:
`ConfigFilepath` seems better self explanatory.
I will update that !